src/pipx/main.py

```diff
@@ -342,7 +346,8 @@ def _add_install(subparsers: argparse._SubParsersAction) -> None:
         default=DEFAULT_PYTHON,
         help=(
             "The Python executable used to create the Virtual Environment and run the "
-            "associated app/apps. Must be v3.6+."
+            "associated app/apps. The Python version for the Python Launcher. "
+            "Must be v3.6+."
         ),
```

Comment:
The message reads like a salad of words and not really comprehensive to me. Could use a rewrite from scratch. (Same for other similar arguments)

Reply:
Good one, I updated the wording, hopefully this is better :+1: